assets/js/lib/cityDropdown.js

```diff
@@ -0,0 +1,22 @@
+
+let citySelector = document.getElementById('citySelector')
+
+const urlParams = new URLSearchParams(window.location.search);
+const city = urlParams.get('city');
+
+citiesWithResources.forEach(element => {
+    var option = document.createElement("option");
+    option.text = element.city;
+    citySelector.add(option)
+});
+
+if (city != undefined) {
+    citySelector.value = city
+}
+
+citySelector.addEventListener("change", function () {
+    var base_url = window.location.origin;
```

Comment:
This needs to be 

```js
let base_url = window.location.origin + window.location.pathname;
```

Otherwise, it doesn't work correctly on local envs with `file://...`